cloud-tests-infrastructure/readme.md

```diff
@@ -1,5 +1,5 @@
 # Cloud tests infrastructure
 
-Cloud resources used in [LeanCode.Azure.Tests](test/Infrastructure/LeanCode.Azure.Tests/LeanCode.Azure.Tests.csproj)
+Cloud resources used in [LeanCode.Azure.Tests](../test/Infrastructure/LeanCode.Azure.Tests/LeanCode.Azure.Tests.csproj) and [LeanCode.AuditLogs.Tests](../test/Infrastructure/LeanCode.AuditLogs.Tests/LeanCode.AuditLogs.Tests.csproj)
```

Comment:
I'm not sure we should have a separate project for these tests. `LeanCode.Azure.Tests` seems sufficient to contain both of them.